tests/RobotFramework/tests/cumulocity/registration/device_registration.robot

```diff
@@ -161,6 +161,15 @@ Register tedge-log-plugin when tedge-mapper-c8y is not running #2389
     Should Have MQTT Messages    te/device/offlinechild3///cmd/log_upload/+
 
 
+Auto register disabled
+    Execute Command    sudo tedge config set c8y.entity_store.auto_register false
+    Restart Service    tedge-mapper-c8y    
+    Service Health Status Should Be Up    tedge-mapper-c8y    
+    Execute Command    sudo tedge mqtt pub 'te/device/${CHILD_SN}///a/temperature_high' '{ "severity": "critical", "text": "Temperature is very high", "time": "2021-01-01T05:30:45+00:00" }' -q 2 -r
```

Comment:
```suggestion
    Execute Command    sudo tedge mqtt pub 'te/device/new_device///a/temperature_high' '{ "severity": "critical", "text": "Temperature is very high", "time": "2021-01-01T05:30:45+00:00" }' -q 2 -r
```
Future proofing, as that `CHILD_SN` will appear as already registered once we implement persistent entity store. This makes it clearer, that it's a "'previously unseen" device.